### DISCUSSION TITLE
Remote Start/stop of sdrtrunk channels

### DISCUSSION THREAD

**Discussion by hoffadg** [chuot/rdio-scanner#305] [Q&A] [answered]
Is it possible to automate the start/stop of a channel in sdrtrunk so it is only on and scanning when I'm using rdio scanner?  Or do most people just let sdrtrunk run continuously?  For me sdrtrunk runs on an old windows pc in my basement so I've been running down there to start it every time I want to use rdio scanner on my phone.

**Comment by chuot** [comment, marked as answer]
There is no such plan. It's best to leave SDRTrunk running all the time.
